test/with_dummyserver/test_poolmanager.py

```diff
@@ -312,6 +312,12 @@ def test_http_with_ca_cert_dir(self):
             r = http.request("GET", "http://%s:%s/" % (self.host, self.port))
             assert r.status == 200
 
+    @pytest.mark.parametrize("char", [" ", "\r", "\n", "\x00"])
+    def test_invalid_method_not_allowed(self, char):
+        with pytest.raises(ValueError):
+            with PoolManager() as http:
+                http.request("GET" + char, "/")
+
```

Comment:
For what it's worth, I think I would have left out this test and considered that the connection pool test is enough

Reply:
That's true, I'll do that. :)